backend/app.py

```diff
@@ -0,0 +1,27 @@
+import io
+
+from src.config import DATA_DIR_PATH
+from src.File.FileManager import create_unique_filename
+from src.Logging.Logging import logger
+from src.Web.GoogleCloudStorage import upload_file
+from src.Web.WebCrawler import (
+    ContentExtractor,
+    LinkResolver,
+    SessionManager,
+    WebCrawler,
+)
+
+if __name__ == "__main__":
+    start_url = "https://admissions.ucsc.edu/"
```

Comment:
this is fine for now, but ideally it should start at ucsc.edu. that being said, correct me if I'm wrong, but would it just jump back to ucsc.edu anyway?